src/content/docs/data-apis/ingest-apis/event-api/incident-event-rest-api.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: Incident event REST API
+tags:
+  - Alerts and Applied Intelligence
+  - Applied intelligence
+  - Incident intelligence
+metaDescription: Read about how to report activities from your incident management system to New Relic for advanced correlation.
+---
+
+You can use the incident event REST API to report custom events to New Relic.
```

Comment:
I wanted to confirm that they want to call this this name. Is it definitely a 'REST API'? I want to confirm that because my current understanding reading this is that it's a REST API that then sends data to our Event API. Am I getting that correct? if so, I wonder if we might make that more clear/explicit.

Reply:
Yeah, that's an important distinction. Good call.